### PR TITLE
fix broken links due to docs updates

### PR DESCRIPTION
**Pull request by cassiebreviu**
**Description**:  There was a reorg/update of docs and the links changed. This is a PR to update the links to reflect the changes in the master branch.

### CODE REVIEW

File: onnxruntime/test/platform/ios/ios_package_test/README.md [hunk 1]
```diff
@@ -14,15 +14,15 @@ The iOS End-to-End Test App will use CocoaPods to install the Onnx Runtime C/C++
 ### Model used
 - [sigmoid ONNX model](https://github.com/onnx/onnx/blob/f9b0cc99344869c246b8f4011b8586a39841284c/onnx/backend/test/data/node/test_sigmoid/model.onnx) converted to ORT format
 
-    Here's the [document](http://www.onnxruntime.ai/docs/how-to/deploy-on-mobile.html#1-create-ort-format-model-and-configuration-file-with-required-operators) about how you can convert an ONNX model into ORT format.
+    Here's the [document](onnxruntime.ai/docs/tutorials/mobile/) about how you can convert an ONNX model into ORT format.
```
Comment (edgchen1) [inline onnxruntime/test/platform/ios/ios_package_test/README.md:17]: maybe this one instead?
https://onnxruntime.ai/docs/tutorials/mobile/model-conversion.html

File: docs/python/inference/notebooks/onnxruntime-nuphar-tutorial.ipynb [hunk 2]
```diff
@@ -2,191 +2,197 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
```
Comment (edgchen1) [inline docs/python/inference/notebooks/onnxruntime-nuphar-tutorial.ipynb:5]: lots of changes in this file, perhaps some were unintentional?

Comment (cassiebreviu) [reply docs/python/inference/notebooks/onnxruntime-nuphar-tutorial.ipynb]: reverted notebook to remove changes from this pr

File: onnxruntime/test/platform/ios/ios_package_test/README.md (hunk 1 above)
Comment (cassiebreviu) [reply onnxruntime/test/platform/ios/ios_package_test/README.md]: updated! :)

**Review by edgchen1 — APPROVED** [review]

**Review by raghavthind2005 — APPROVED** [review]

**Pull request closed without merging**